pkgs/servers/dict/gcide/default.nix

```
@@ -0,0 +1,106 @@
+{
+  autoreconfHook,
+  bison,
+  dict,
+  fetchurl,
+  flex,
+  lib,
+  libmaa,
+  stdenv,
+}:
+
+let
+  sed-scripts = {
+    check = ./sed-scripts/check.sed;
+    fixes = ./sed-scripts/fixes.sed;
+    post_webfilter = ./sed-scripts/post_webfilter.sed;
+  };
+
+  _major_debver = "0.48";
+  _debver = "${_major_debver}.5+nmu4";
```

```suggestion
  major_debver = "0.48";
  debver = "${major_debver}.5+nmu4";
```
those are already private